nuxeo-features/nuxeo-platform-relations/nuxeo-platform-relations-core-listener/src/test/java/org/nuxeo/ecm/platform/relations/core/listener/TestPublishRelationsListener.java

```diff
@@ -44,12 +43,13 @@
 import org.nuxeo.ecm.platform.relations.api.impl.ResourceImpl;
 import org.nuxeo.ecm.platform.relations.api.impl.StatementImpl;
 import org.nuxeo.ecm.platform.relations.api.util.RelationConstants;
+import org.nuxeo.ecm.platform.test.PlatformFeature;
 import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
 
 @RunWith(FeaturesRunner.class)
-@Features(CoreFeature.class)
+@Features(PlatformFeature.class)
```

Comment:
Can you revert this change? It doesn't looks like linked to your fix.

Reply:
It is, the test relies on comments

Reply:
This is necessary because we call the CommentManager which now requires the `PlatformFeature` to work in tests